docs/docs/reference/new-types/type-lambdas-spec.md

```diff
@@ -94,6 +87,20 @@ is treated as a shorthand for
 ```scala
 [F >: Nothing <: [X] =>> Coll[X]]
 ```
+Abstract types and opaque type aliases remember the variances they were created with. So the type
+```scala
+def F2[-A, +B]
```

Comment:
I think this should be **`type`**`F2[-A, +B]` is known to be **contra**variant in `A` and **co**variant in `B`

Reply:
Thanks! Fixed.